### PR TITLE
docs: update root readme file

### PR DESCRIPTION
**Pull request by francescosave**
This replaces the previous pull request.
https://github.com/TeleCO2gram/backend/pull/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,63 @@
...
+<br />
+<div align="center">
+  <a href="https://github.com/TeleCO2gram/backend">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+<h3 align="center">TelegramCO2Monitor Backend</h3>
+
+  <p align="center">
+    TelegramCO2Monitor is a backend of App monitor telegram Co2 emissions.
```
Comment (larafenny) [inline README.md:13]: TeleCO2gram

**Review by larafenny — APPROVED** [review]
There is a small detail to fix. The project name is TeleCO2gram. This is a draft in continuous update, so for me, this fix can be done in the future.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,63 @@
...
+[license-url]: https://github.com/TeleCO2gram/backend/blob/main/license.txt
+
+
+[Python]: https://img.shields.io/badge/Python-326c9d?style=for-the-badge&logo=python&logoColor=61DAFB
+[Python-url]: https://python.org/
+
+[FastAPI]: https://img.shields.io/badge/FastAPI-05988a?style=for-the-badge&logo=fastapi&logoColor=61DAFB
+[FastAPI-url]: https://fastapi.tiangolo.com/
+
+[Bootstrap.com]: https://img.shields.io/badge/Bootstrap-563D7C?style=for-the-badge&logo=bootstrap&logoColor=white
```
Comment (larafenny) [inline README.md:62]: Not used

Comment (francescosave) [reply README.md:62]: pinned in issues #8

File: README.md (hunk 1 above)
Comment (francescosave) [reply README.md:13]: pinned in issues #7

**Pull request merged**